dexios/src/cli.rs

```diff
@@ -69,12 +69,10 @@ pub fn get_matches() -> clap::ArgMatches {
                 .help("Force all actions"),
         )
         .arg(
-            Arg::new("aead")
```

Comment:
You should add 'deoxys' argument if `deoxis-ii-256` feature is enabled. Or remove deoxys feature https://github.com/brxken128/dexios/blob/master/dexios/Cargo.toml#L21

Reply:
> The deoxys feature isn't enabled for the CLI app though, is it? > I'm not too sure I see the point in making a switch for a feature that isn't enabled.

You can enable deoxys if you run cli with `--features deoxys-ii-256`

Reply:
> Until performance of Deoxys improves, I'm not comfortable adding the option to encrypt with it back in though.

ok 👌🏼